apps/api/src/modules/lineups/lineups.service.ts

```diff
@@ -0,0 +1,204 @@
+import { pool } from "../../database";
+
+type Role = "COACH" | "PLAYER";
+
+export type Position =
+  | "GK"
+  | "RB"
+  | "CB"
+  | "LB"
+  | "RWB"
+  | "LWB"
+  | "CDM"
+  | "CM"
+  | "CAM"
+  | "RW"
+  | "LW"
+  | "ST"
+  | "CF"
+  | "UNASSIGNED";
+
+export async function createLineup(input: {
```

Comment:
This function is doing ALOT and you should consider breaking out helpers functions like assertCoachOwnsFixture, assertValidPlayers, replaceLineupPlayers would make it easier to follow and the code much more maintainable.

One part about making "self-documenting" code is by breaking things up into readable and digestible chunks